pkg/driver/driver.go

```diff
@@ -41,6 +41,7 @@ type Driver struct {
 	storageProviders map[string]storageprovider.StorageProvider
 	flavor           flavor.Flavor
 	grpc             NonBlockingGRPCServer
+	nfsPodMonitor    *kubernetes.Monitor
```

Comment:
This is now starting to bleed k8s constructs into our driver.  The driver should remain agnostic.  We should continue to put this type of stuff into our flavor.

Reply:
ah..looks like i forgot to clean this up. This is not used and moved to Flavor instead.
```// Flavor of the CSI driver
type Flavor struct {
	crdClient  *crd_client.Clientset
	nodeName   string
	kubeClient kubernetes.Interface

	claimInformer cache.SharedIndexInformer
	claimIndexer  cache.Indexer
	claimStopChan chan struct{}

	eventRecorder record.EventRecorder
	nfsMonitor    *Monitor
}```